.gitignore

```diff
@@ -118,4 +118,8 @@ dist
 .yarn/install-state.gz
 .pnp.*
 
+# ignore lock files of other package managers
+pnpm-lock.yaml
```

Comment:
Should we also remove `pnpm-lock.yaml` since we start ignoring it?

Reply:
Sure, I was not aware there is one already. This is what this change aims to avoid because for sure some lockfile would be outdated if we use multiple ones.